### PR TITLE
Implemented persistent-single-player run strategy returns the latest use...

### PR DESCRIPTION
**Pull request by jaimedp**
...r run or a fresh one if non exists

### CODE REVIEW

File: src/managers/run-strategies/persistent-single-player-strategy.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    getRun: function () {
+        return this._getAllRuns()
+            .then(this._loadAndCheck);
+    },
+
+    _getAllRuns: function () {
+        var session = JSON.parse(this._store.get(keyNames.EPI_SESSION_KEY) || '{}');
+        return this.run.query({
+            'user.id': session.userId || '0000',
```
Comment (narenranjit) [inline src/managers/run-strategies/persistent-single-player-strategy.js]: Why default to this instead of leaving null?

Comment (jaimedp) [inline src/managers/run-strategies/persistent-single-player-strategy.js]: I wanted the query string to look 'incorrect' as opposed to just "userId=" which filters out all users, but is doesn't look evil at first glance :)

Comment (narenranjit) [inline src/managers/run-strategies/persistent-single-player-strategy.js]: you should've defaulted to 666 then :)

**Comment by narenranjit** [comment]
@jaimedp I can't merge this from the web ui, can you just merge into master after resolving conflicts?

**Comment by jaimedp** [comment]
@narenranjit fixed merge conflicts

**Pull request merged**